app/controllers/api/mixins/service_templates.rb

```diff
@@ -17,6 +19,14 @@ def order_service_template(id, data, scheduled_time = nil)
       def service_template_ident(st)
         "Service Template id:#{st.id} name:'#{st.name}'"
       end
+
+      def api_request_allowed?
+        standalone_api_request? ? !Settings.product.deny_api_service_ordering : true
```

Comment:
Can the Setting have a positive name?

Reply:
The only issue I have with changing it to a positive name is then the users who want it to work the way it works after this PR would be to set the value to "false", which feels weird but I don't know if we have a precedent for this or not.

Reply:
No, but I think the default should be in [here](https://github.com/ManageIQ/manageiq/blob/master/config/settings.yml) anyway.